src/components/CreditCardDetails/CreditCardDetailsService.js

```diff
@@ -0,0 +1,16 @@
+export const ACTIONS = {
+  TOGGLE_INFO: 'toggle info'
+};
+
+export const INITIAL_STATE = {
+  isShowingInfo: false
+};
+
+export const stateReducer = action => prevState => {
+  switch (action) {
+    case ACTIONS.TOGGLE_INFO:
+      return { isShowingInfo: !prevState.isShowingInfo };
+    default:
+      return prevState;
+  }
+};
```

Comment:
I know this is a redux-y way to handle this, but I don't see the benefit of this indirection as opposed to simply updating the state inside the component when you have literally one action...aaaaand especially as this doesn't have any tests 😝